### ISSUE TITLE
Render view to string with precompilation

### ISSUE THREAD

**Issue by pebezo** [aspnet/MvcPrecompilation#189]
Hi,

I'm using this sample to render a view to a string: 

https://github.com/aspnet/Entropy/blob/dev/samples/Mvc.RenderViewToString/RazorViewToStringRenderer.cs 

This works fine in development (running it via VS), but if I publish the project and run it under IIS it complains that it cannot find the view. In this scenario, the published folder does not contain the cshtml files, but it does have ProjectName.PrecompiledViews.dll.

If I publish without precompilation (basically remove ```<PackageReference Include="Microsoft.AspNetCore.Mvc.Razor.ViewCompilation" Version="2.0.0" PrivateAssets="All" />``` from .csproj) and deploy to the same server then it works.

I'm using ASP.NET Core 2.0 (under .NET Framework 4.7).

**Comment by pranavkm** [comment]
@pebezo you're likely running into a variant of https://github.com/aspnet/Mvc/issues/6676. Could you specify the value what you're passing in to `FindView`?

**Comment by pebezo** [comment]
Yes, sounds like that issue. I'm passing in "FolderName/ViewName" (without any extensions). I've tried may variations like prepending "~/" or "~/Views" and/or with extensions. The only one that worked was "FolderName/ViewName".

**Comment by pranavkm** [comment]
We'll use the other issue to track further work in this area if any. For now

(a) Use an extension when specifying a path i.e. `/Views/FolderName/ViewName.cshtml`
(b) Use `GetView` to look up paths.

We'll update the RenderViewToString sample to include the correct usage so this is clearer for future users.
